com.playeveryware.eos/Runtime/Core/EOS_SDK_Additions/EOSClientCredentials.cs

```diff
@@ -45,13 +45,18 @@ static EOSClientCredentials()
         }
 
         public EOSClientCredentials()
+        {
+            EncryptionKey = GenerateEncryptionKey();
```

Comment:
Does this override the serialized value? Make sure it does not.

Reply:
It does not override the serialized value - because there is a specific constructor that is used for deserializing from JSON, it's specified using the `[JsonConstructor]` attribute.

Marking conversation as resolved.